mongodb_migrations/config.py

```diff
@@ -14,17 +14,19 @@ class Configuration(object):
     mongo_host = '127.0.0.1'
     mongo_port = '27017'
     mongo_url = ''
-    mongo_database = ''
+    mongo_database = ''    
+    mongo_auth_user = ''
```

Comment:
if you don't mind, let's go with just `username` and `password`

Reply:
sure :)